etc/profile-a-l/avidemux3_qt5.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for avidemux3_qt5
+# Description: The GUI for Avidemux using the QT5 framework.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include avidemux3_qt5.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+# Additional whitelisting
+whitelist /usr/share/avidemux3 # permit translation packages
```

Comment:
It redirects to avidemux3_qt5.profile to inherit the Qt dependencies. Should this be pointed out more explicitly in the redirect comment?

Reply:
@hknaack [on Dec 12](https://github.com/netblue30/firejail/pull/5522#discussion_r1046397511):

> It redirects to avidemux3_qt5.profile to inherit the Qt dependencies. Should
> this be pointed out more explicitly in the redirect comment (I had it in the
> commit message)?

Sorry, I should have looked better.  I assumed that all redirects were to
`avidemux.profile`.

When looking at just `avidemux3_jobs_qt5.profile` in isolation, I think that
the redirect is probably clear enough as is.

Aside: Profile-related PRs in this repository often do not contain much
information (if any) in the commit messages and they are usually merged faster
than other PRs, so I personally usually tend to focus on "speed" rather than on
quality when reviewing such PRs, to avoid missing the chance to review them
before they are merged.  Anyway, this is a process issue; informative commit
messages are appreciated.